src/App.jsx

```diff
@@ -17,7 +17,8 @@ const App = () => {
           <Route path="/aboutus" exact component={AboutUs} />
           <Route path="/now-u-app" exact component={NowUApp} />
           <Route path="/get-in-touch" exact component={GetInTouch} />
-          <Redirect to={'/'} />
+          <Route path="/error404" exact component={err404} />
+          <Redirect to={'/error404'} />
```

Comment:
I think its probably a bit neater to remove the redirect and just match on any route.
This can be done by just having this instead of the final 2 lines in the switch (you will also need to remove the redirect import)
```suggestion
          <Route exact component={err404} />
```

Reply:
This also keeps the domain of the user when they get the 404 which I think is quite a good thing?